inference-engine/src/gna_plugin/gna_model_serial.hpp

```diff
@@ -27,21 +27,21 @@
  */
 #if GNA_LIB_VER == 2
 #define HEADER_MAJOR 2
-#define HEADER_MINOR 1
+#define HEADER_MINOR 2
 #else
 #define HEADER_MAJOR 1
 #define HEADER_MINOR 2
 #endif
 
 
 /**
- * @brief Header version 2.1
+ * @brief Header version 2.2
  */
 struct ModelHeader {
     /**
      *@brief MagicNumber – GNAM in ascii table, equals to hex 0x474e414d
      */
-    char gnam[4];
+    char gnam[4] = {0, 0, 0, 0};
```

Comment:
either char gnam[4] = {}; or char gnam[] = {'\0', '\0', '\0','\0'}; as '\0' is a correct character literal